packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1230,7 +1322,7 @@ export class Combobox
     this.updateItemProps();
 
     this.selectedItems = this.getSelectedItems();
-    this.filteredItems = this.getFilteredItems();
+    this.filteredItems = this.visibleKeyboardNavItems;
```

Comment:
If `filteredItems` and `visibleItems` are the same, can we just use `filteredItems`? Make `filteredItems` a getter with the `visibleKeyboardNavItems` logic?

```ts
@property() get filteredItems():  HTMLCalciteComboboxItemElement["el"][]  {
  return this.keyboardNavItems.filter((item) => !isHidden(item));
}
```